tools/schemas/enclave-config.schema.json

```diff
@@ -452,6 +452,11 @@
           "description": "",
           "default": true
         },
+        "unsafe_host_signals": {
+          "type": "boolean",
+          "default": true,
```

Comment:
Should this eventually become false? Is there an issue that tracks which tests break currently in that case?

Reply:
Not yet. We should have a larger discussion about signal handling with SGX1. I will open an issue and refer to `unsafe_host_signals` there.

Reply:
https://github.com/lsds/sgx-lkl/issues/680